extension.json

```diff
@@ -118,7 +118,28 @@
             "type": "boolean"
           },
           "value": {
-            "type": "string"
+            "anyOf": [
```

Comment:
@dompuiu How do you feel about leaving this one not as "oneOf" at the top level? There are a lot of properties to duplicate in both blocks, and in terms of reading it as natural language, this one seems to flow okay.

Reply:
I think there is a problem with the top-level `valueIsRegex`. You will be able to send an object with 2 `valueIsRegex` keys. One on the top level and one inside the array.